pkg/manifests/manifests.go

```diff
@@ -1769,6 +1783,14 @@ func (f *Factory) PrometheusAdapterDeployment(apiAuthSecretName string, requesth
 		},
 	)
 
+	if auditLogsEnabled(config.AuditLevel) {
+		lvl := strings.ToLower(config.AuditLevel)
+		spec.Containers[0].Args = append(spec.Containers[0].Args,
+			fmt.Sprintf("--audit-policy-file=/etc/adapter/audit-policy-%s.yaml", lvl),
+			"--audit-log-path=/tmp/audit.log",
```

Comment:
Do you want to set a maxage or maxsize for this to ensure that we do not run out of space?

Reply:
I agree ... 2 things 
1. not too sure what we want to set as an upper limit (any thoughts? ) 
1. Please note /tmp is a `tmpfs` mount (in our deployment)

Reply:
I think we should follow what the apiserver is doing and have an initContainer setup the directory on the node that we would then use as a volume to store the logs: https://github.com/openshift/cluster-openshift-apiserver-operator/blob/ff289ebf4cd2df621801dd6b76a932ac39f1833b/bindata/v3.11.0/openshift-apiserver/deploy.yaml#L46-L61

Reply:
@simonpasquier I was basing this on https://github.com/sthaha/cluster-monitoring-operator/blob/b728aa6c986e51c4fea9f1111038fb2b3af7a723/assets/prometheus-adapter/deployment.yaml#L60

Reply:
Using `hostPath` is probably problematic security-wise since IIUC it imposes to run as root:
https://github.com/openshift/cluster-openshift-apiserver-operator/commit/95e60ab3a8a9f20a53d523e43d71820c91f55f99

Reply:
good point, do you know of any alternative we could have? In the end, all the logs are stored on the node's disk under /var/log, and I believe that is what the must-gather also expects.

Reply:
In must-gather scripts the `default` SA has all permission so we should be able to `oc exec|cp` from the adapter pod.

I will create a new mount for `/var/log/adapter/[audit.log]` instead of using `/tmp` 

will set the max log size to 100MB, or  do you want that to be configurable  as well @simonpasquier , @dgrisonnet ?